blocks/README.md

```diff
@@ -40,3 +40,27 @@ blocks/
 ```
 
 Please refer to the official [BEM](https://en.bem.info/methodology/quick-start/#introduction) or [BEM Nested](https://en.bem.info/methodology/quick-start/#file-structure) documentation for details if needed.
+
+### Using media queries
+
+For all components, we use the Desktop-first design approach, programming them initially for the desktop version, and subsequently adapting them to breakpoints: `1199px, 659px,` using `max-width` for media queries
```

Comment:
@NazarPauk Great job on updating the docs and including information about the use of media queries! 

I'd like to suggest to rephrase it a bit to improve the understanding of our approach. When mentioning the breakpoints, it's important to clarify that the values `659px` and `1199px` represent the upper limits of the screen sizes targeted by the media queries rather than being breakpoints themselves.

For example, instead of saying _"breakpoints: 1199px, 659px,"_ we could rephrase it to emphasize that these values are the upper limits of our ranges. Something like: _"Our breakpoints are set at` 659px` and `1199px`, representing the upper limits of the screen sizes targeted by our media queries."_

Reply:
Done✅